library/modules/Textures.cpp

```diff
@@ -30,6 +31,10 @@ static std::unordered_map<TexposHandle, long> g_handle_to_texpos;
 static std::unordered_map<TexposHandle, SDL_Surface*> g_handle_to_surface;
 static std::mutex g_adding_mutex;
 
+// it is SDL_PixelFormatEnum::SDL_PIXELFORMAT_RGBA32
+// initialized inplace to avoid including SDL_pixels.h
```

Comment:
Is there a problem with including SDL_pixels.h? We already include SDL_surface.h, so it's not like we're adding a dependency (or is it?)